pkg/collector/target.go

```diff
@@ -28,15 +28,17 @@ type Target struct {
 	metrics            *tree
 	stringValueMapping map[string]map[string]int
 	stopCh             chan struct{}
+	reconnect          bool
 	maxReads           int
 }
 
-func newTarget(tconf *config.Target, stringValueMapping map[string]map[string]int) *Target {
+func newTarget(tconf *config.Target, stringValueMapping map[string]map[string]int, reconnect bool) *Target {
 	t := &Target{
 		address:            fmt.Sprintf("%s:%d", tconf.Hostname, tconf.Port),
 		paths:              tconf.Paths,
 		metrics:            newTree(),
 		stringValueMapping: stringValueMapping,
+		reconnect:          reconnect,
```

Comment:
reconnect is never used.

Reply:
? It is checked on line 148 in target.go.

Reply:
Ok, didn't see it.